examples/step-77/doc/results.dox

```diff
@@ -0,0 +1,92 @@
+<h1>Results</h1>
+
+When running the program, you get output that looks like this:
+@code
+@endcode
+
+The way this should be interpreted is most easily explained by looking at
+the first few lines of the output on the first mesh:
+@code
+Mesh refinement step 0
+  Target_tolerance: 0.0001
+
+  Computing residual vector... norm=0.231202
+  Computing Jacobian matrix
+  Factorizing Jacobian matrix
+  Solving linear system
+  Computing residual vector... norm=0.231202
+  Computing residual vector... norm=0.171585
+  Solving linear system
+  Computing residual vector... norm=0.171585
+  Computing residual vector... norm=0.127245
+  Computing residual vector... norm=0.0796471
+  ...
+@endcode
+What is happening is this:
+- In the first residual computation, KINSOL computes the residual to see whether
+  the desired tolerance has been reached. The answer is no, so it requests the
+  user program to compute the Jacobian matrix (and the function then also
+  factorizes the matrix via SparseDirectUMFPACK).
+- KINSOL then instructs us to solve a linear system of the form
+  $J_k \, \delta U_k = -F_k$ with this matrix and the previously computed
+  residual vector.
+- It is then time to determine how far we want to go in this direction,
+  i.e., do line search. To this end, KINSOL requires us to compute the
+  residual vector $F(U_k + \alpha_k \delta U_k)$ for different step lengths
+  $\alpha_k$. For the first step above, it finds an acceptable $\alpha_k$
+  after two tries, the second time around it takes three tries.
+- Having found a suitable updated solution $U_{k+1}$, the process is
+  repeated except now KINSOL is happy with the current Jacobian matrix
+  and does not instruct us to re-build the matrix and its factorization.
+
+The program also writes the solution to a VTU file at the end
+of each mesh refinement cycle, and it looks as follows:
+<table width="60%" align="center">
+  <tr>
+    <td align="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-77.solution.png" alt="">
```

Comment:
The picture has not been uploaded yet.

Reply:
Correct -- but now done. I'll have to re-generate this once we figure out the issue with the hanging nodes...